tests/data/metadata/modelprogram_meta.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="utf-8"?>
+<rdf:RDF
+  xmlns:dc="http://purl.org/dc/elements/1.1/"
+  xmlns:hsterms="https://www.hydroshare.org/terms/"
+  xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#"
+  xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+>
+  <hsterms:ModelProgramAggregation rdf:about="https://www.hydroshare.org/resource/526e62aac1044285abc2cae4e2f463da/data/contents/modelprogram_resmap.xml#aggregation">
+    <hsterms:modelOperatingSystem>linux</hsterms:modelOperatingSystem>
+    <dc:title>setup againas</dc:title>
+    <hsterms:modelWebsite rdf:resource="https://www.hydroshare.org"/>
+    <hsterms:modelReleaseNotes rdf:resource="https://www.hydroshare.org/resource/526e62aac1044285abc2cae4e2f463da/data/contents/setup.cfg"/>
```

Comment:
Should modelReleaseNotes, modelDocumentation, modelSoftware, and modelEngine be repeatable?